readme.md

```diff
@@ -54,6 +54,7 @@ SFML is an open-source project, and it needs your help to go on growing and impr
 -   Stefan Schindler (tank@sfml-dev.org)
 -   Artur Moreira (artturmoreira@gmail.com)
 -   Vittorio Romeo (vittorioromeo@sfml-dev.org)
+-   Chris Thrasher (chrisjthrasher@gmail.com)
```

Comment:
Not sure we should allow this change, could be API breaking

Reply:
But it increases test coverage by nearly a whole percentage point

<img width="579" alt="Screen Shot 2022-09-27 at 4 43 32 PM" src="https://user-images.githubusercontent.com/39244355/192650507-735708ec-2868-45f5-a790-5c9e931c1ef8.png">

Reply:
Well I'm sold. Clearly the a good choice for SFML given the hard work 😄